tools/webdriver/webdriver/transport.py

```diff
@@ -27,17 +29,12 @@ def from_http_response(cls, http_response):
         # >       "application/json; charset=utf-8"
         # >    "cache-control"
         # >       "no-cache"
-        assert http_response.getheader("Content-Type") == "application/json; charset=utf-8"
-        assert http_response.getheader("Cache-Control") == "no-cache"
```

Comment:
Sometimes it sends plain text as an error message.

But in any case that's not what this tells us. This tells us that they set the headers as specified. It's quite possible to be sending json with the wrong headers (especially Cache-Control). There's just no way every test should fail if this is wrong vs the test for this precise thing failing.